js_modules/dagit/packages/core/src/resources/ResourceRoot.tsx

```diff
@@ -0,0 +1,208 @@
+import {gql, useQuery} from '@apollo/client';
+import {
+  Box,
+  Colors,
+  Heading,
+  Page,
+  PageHeader,
+  SplitPanelContainer,
+  Table,
+  Tag,
+  Tooltip,
+} from '@dagster-io/ui';
+import * as React from 'react';
+import {useParams} from 'react-router-dom';
+import styled from 'styled-components/macro';
+
+import {PYTHON_ERROR_FRAGMENT} from '../app/PythonErrorFragment';
+import {useTrackPageView} from '../app/analytics';
+import {useDocumentTitle} from '../hooks/useDocumentTitle';
+import {RepositoryLink} from '../nav/RepositoryLink';
+import {SidebarSection} from '../pipelines/SidebarComponents';
+import {Loading} from '../ui/Loading';
+import {repoAddressToSelector} from '../workspace/repoAddressToSelector';
+import {RepoAddress} from '../workspace/types';
+
+import {ResourceRootQuery} from './types/ResourceRoot.types';
+
+interface Props {
+  repoAddress: RepoAddress;
+}
+
+const remapName = (inName: string): string => {
+  if (inName === 'StringSourceType') {
+    return 'String';
+  } else if (inName === 'IntSourceType') {
+    return 'Int';
+  } else if (inName === 'BoolSourceType') {
+    return 'Bool';
+  }
+  return inName;
+};
+
+export const ResourceRoot: React.FC<Props> = (props) => {
+  useTrackPageView();
+
+  const {repoAddress} = props;
+
+  const {resourceName} = useParams<{resourceName: string}>();
+
+  useDocumentTitle(`Resource: ${resourceName}`);
+
+  const resourceSelector = {
+    ...repoAddressToSelector(repoAddress),
+    resourceName,
+  };
+  const queryResult = useQuery<ResourceRootQuery>(RESOURCE_ROOT_QUERY, {
+    variables: {
+      resourceSelector,
+    },
+    fetchPolicy: 'cache-and-network',
+    partialRefetch: true,
+    notifyOnNetworkStatusChange: true,
+  });
+
+  return (
+    <Page>
+      <PageHeader
+        title={
+          <Box flex={{direction: 'row', alignItems: 'center', gap: 12}}>
+            <Heading>{resourceName}</Heading>
+          </Box>
+        }
+      />
+      <Loading queryResult={queryResult} allowStaleData={true}>
+        {({topLevelResourceOrError}) => {
+          if (topLevelResourceOrError.__typename !== 'TopLevelResource') {
+            return null;
+          }
+
+          const configuredValues = Object.fromEntries(
+            topLevelResourceOrError.configuredValues.map((cv) => [cv.key, cv.value]),
+          );
+
+          return (
+            <SplitPanelContainer
+              identifier="explorer"
+              firstInitialPercent={70}
+              firstMinSize={400}
+              height="calc(100% - 60px)"
```

Comment:
Not sure this new prop is necessary -- are there some other `SplitPanelContainer` uses that look similar? I think maybe a wrapper div to give the element height might do the job here. `OpsRoot` perhaps?

Reply:
Hmm, it looks like the split panel container doesn't want to fill the parent in that case. You can see that the parent div has 100% height but the container itself doesn't expand:

<img width="895" alt="Screen Shot 2023-02-02 at 3 07 08 PM" src="https://user-images.githubusercontent.com/10215173/216471333-cdbb00c4-db5e-46a7-a916-893b9e8f23cf.png">